lib/harmony/harmonytask/taskTypeHandler.go

```diff
@@ -0,0 +1,274 @@
+package harmonytask
+
+import (
+	"context"
+	"errors"
+	"io"
+	"net/http"
+	"strconv"
+	"sync/atomic"
+	"time"
+
+	"github.com/filecoin-project/lotus/lib/harmony/harmonydb"
+	logging "github.com/ipfs/go-log/v2"
+)
+
+var logger = logging.Logger("harmonytask")
```

Comment:
Nit: We usually call the logger instance `log`

```suggestion
var log = logging.Logger("harmonytask")
```